packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -105,6 +106,17 @@ describe.each([
       }
     }
 
+    private async performSearchFullResponse(): Promise<SearchResponse<Row>> {
+      if (isInMemory) {
+        return { rows: dataFilters.search(_.cloneDeep(rows), this.query) }
+      } else {
+        return config.api.row.search(table._id!, {
+          ...this.query,
+          tableId: table._id!,
+        })
+      }
+    }
```

Comment:
I think it would be better to delete this and make the other performSearch function return the full response and then pull the rows out in each of the places where it's called. I don't think it makes sense to have two of these that are almost entirely identical.